aws/database/dynamodb/scripts/contact_details_table/scan_and_patch_new_phone_numbers.py

```diff
@@ -1,14 +1,9 @@
 from dataclasses import dataclass
 
-from mypy_boto3_dynamodb.service_resource import Table
```

Comment:
If we remove this then I don't think the script would work as expected

Reply:
It was an unused import - the table variable wasn't annotated with mypy
mypy is just for type annotations, doesn't change functionality